metadata-ingestion/src/datahub/ingestion/source/dbt.py

```diff
@@ -36,6 +36,7 @@ class DBTConfig(ConfigModel):
     manifest_path: str
     catalog_path: str
     env: str = "PROD"
+    target_platform: str = "dbt"
```

Comment:
I'm a bit unsure on this config being optional - when does it make sense to use "dbt" as the platform?

Maybe we can have `target_platform` required and then also add a `load_schemas` flag to the config?